litgpt/chat/base.py

```diff
@@ -110,6 +110,51 @@ def decode(fabric: L.Fabric, tokenizer: Tokenizer, token_stream: Iterator[torch.
     return tokens_generated
 
 
+def process_prompt(prompt, model, tokenizer, prompt_style, fabric, temperature, top_k, top_p, stop_tokens):
+    if not prompt.lower().strip() in ("", "quit", "exit"):
```

Comment:
Why is this necessary if you are also checking it in L152?

If it's really necessary, I would add the nit that it should do an early return to avoid the extra indentation level for the whole code block

Reply:
Good callout. I cleaned this up.